core/src/main/java/jenkins/model/GlobalUIConfiguration.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * The MIT License
+ * 
+ * Copyright (c) 2012, Dominik Bartholdi
```

Comment:
Welcome to the future! It's 2020 ;) 
And last time I checked your work should be attributed to CB, not to Dominik :)